Gemfile

```diff
@@ -9,16 +9,31 @@ path "./" do
   gem "refinerycms-resources"
 end
 
-gem 'quiet_assets'
 gem 'spring'
 gem 'spring-commands-rspec'
 gem 'poltergeist', '>= 1.8.1'
 
 # Add support for refinerycms-acts-as-indexed
 gem 'refinerycms-acts-as-indexed', ['~> 3.0', '>= 3.0.0']
 
+
+###########################
+# Rails 5 temp. gem sources
+
+gem 'refinerycms-i18n', git: 'https://github.com/refinery/refinerycms-i18n', branch: 'feature/rails-5'
+
+group :development do
+  gem 'listen', '~> 3.0'
+end
+
+#
+###########################
```

Comment:
I will check that. But until we don't merge this PR https://github.com/refinery/refinerycms-i18n/pull/65 with a version bump, i will have to specify this source